tools/create-openbsd-gce-ci.sh

```diff
@@ -67,7 +67,7 @@ EOF2
     set -eux
     ulimit -d 8000000
     export GOCACHE=/syzkaller/go-cache
-    mkdir -p $GOCACHE
+    mkdir -p \$GOCACHE
```

Comment:
You could instead turn this into a literal heredoc, removing the need to escape variables.

```sh
<<-'EOF'
```

Reply:
There's another case of `\$` above and it's inside a double heredoc'd region. I'm feeling nervous about making more changes without having a linux machine to this test on. The VMs I have access to aren't kvm-capabale, so it's an extra hurdle...

Reply:
Does my explanation make sense? I'd like to merge this.